src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ObjectWriter.cs

```diff
@@ -318,12 +350,17 @@ public void BuildCFIMap(ObjectNode node)
 
                 if (_targetPlatform.OperatingSystem == TargetOS.Windows)
                 {
-                    SwitchSection(_nativeObjectWriter, "xdata");
-                    EmitAlignment(4);
-
-                    // TODO: Make sure that the same blobs get folded by emitting them to comdat sections
-
                     string blobSymbolName = "_unwind" + (i++).ToStringInvariant() + _currentNodeName;
+
+                    ObjectNodeSection section = ObjectNodeSection.XDataSection;
+                    if (node.ShouldShareNodeAcrossModules(context) && context.Target.OperatingSystem == TargetOS.Windows)
```

Comment:
Sounds reasonable. Can you put a TODO on them in CompilationGroup so that we don't add new dependencies?